extensions/amp-story/amp-story-auto-ads.md

```diff
@@ -16,39 +16,75 @@ limitations under the License.
 
 # <a name="`amp-story-auto-ads`"></a> `amp-story-auto-ads`
 
-
 ## THIS IS A WORK-IN-PROGRESS
 
 <table>
   <tr>
     <td width="40%"><strong>Description</strong></td>
-    <td>FILL THIS IN</td>
+    <td>Dynamically inserts ads into a Story.</td>
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td>FILL THIS IN</td>
+    <td>Experimental</td>
   </tr>
   <tr>
     <td width="40%"><strong>Required Script</strong></td>
-    <td><code>&lt;script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-story-auto-ads-0.1.js">&lt;/script></code></td>
+    <td><code>&lt;script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js">&lt;/script></code></td>
   </tr>
   <tr>
     <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
-    <td>FILL THIS IN</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>Examples</strong></td>
-    <td>FILL THIS IN</td>
+    <td>N/A</td>
   </tr>
 </table>
 
 ## Behavior
+`amp-story-auto-ads` extension dynamically inserts ads (implemented as `amp-ad`)
+into the story at the time when the content is consumed.
```

Comment:
"While content is being consumed by the user".

Reply:
done